aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -182,185 +175,163 @@ image::time-based-aggregator.png[Time based Aggregator]
+| Incremental Aggregation Route | Route | Components chain to execute for every new element that is aggregated. The payload is the list of all the elements that have been aggregated in the aggregation from the first element to the one that is currently aggregated. | |
 |===
 
 ==== Raises
 
 * AGGREGATORS:GROUP_COMPLETED
-+
-When a new element has to be added to an already completed group (and the group was not yet evicted).
-+
+
+An error occurred during an attempt to add a new element to an already completed group that was not yet evicted.
+
 * AGGREGATORS:GROUP_TIMED_OUT
-+
-When a new element has to be added to a group that timed out (and the group was not yet evicted).
-+
+
+An error occurred during an attempt to add a new element to a group that timed out but had not yet been evicted.
+
 * AGGREGATORS:NO_GROUP_ID
-+
-When the expression that resolves to the group ID returns null.
-+
+
+The expression that resolves to the group ID returns null.
+
 * AGGREGATORS:NO_GROUP_SIZE
-+
-When the expression that resolves to the group size returns null.
-+
+
+The expression that resolves to the group size returns null.
+
 * AGGREGATORS:AGGREGATOR_CONFIG
-+
-When the group size or timeout has invalid values (for example, `groupSize < 0`).
-* AGGREGATORS:OBJECT_STORE_ACCESS
-+
-If there is any issue accessing the ObjectStore used to store the aggregated values.
 
----
-image::group-based-aggregator.png[Group based Aggregator]
-. Input Data
-. Aggregated elements (Not complete)
-. Aggregated elements (Complete)
-. Input Data (same as 1) + New Variables
-. Aggregated elements, one of :
-.. `groupSize` reached (same as 3)
-.. `timeout` (only if timed out groups are accepted)
+The Group size or Timeout fields have invalid values, for example, groupSize < 0.
 
+* AGGREGATORS:OBJECT_STORE_ACCESS
 
+An error occurred during an attempt to access the object store used to store the aggregated values.
 
 
 == Sources
 
 [[aggregator-listener]]
 === Aggregator Listener
 
+An *Aggregator listener* is a source for listening to elements triggered by an aggregator scope.
+
 `<aggregators:aggregator-listener aggregatorName="exampleAggregator" includeTimedOutGroups="false">`
 
 [IMPORTANT]
-Listeners are only able to reference aggregators that are inside a flow. If an aggregator is declared in a sub-flow, it will not be visible to listeners.
+An Aggregator listener references only aggregator scopes that are inside a flow. Aggregator scopes declared in a sub-flow are not visible to Aggregator listeners.
+
+Once the aggregator scope referenced by the listener completes an aggregation, the listener is triggered with a list of all the elements. +
+Because the *Aggregator listener* is a source, it is located in a different flow than the aggregator. The listener cannot access the context from the aggregator's flow, and therefore, cannot access the flow's variables.
```

Comment:
```suggestion
Because the *Aggregator listener* is a source, it is located in a different flow than the aggregator. The listener cannot access the context from the aggregator's flow, and therefore cannot access the flow's variables.
```